shared/build.gradle.kts

```diff
@@ -106,4 +130,8 @@ android {
 
 compose {
     kotlinCompilerPluginArgs.add("suppressKotlinVersionCompatibilityCheck=1.9.20")
+}
+
+repositories {
+    maven("https://maven.pkg.jetbrains.space/kotlin/p/wasm/experimental")
```

Comment:
I think we can define only 1 in setting and that should work.

Reply:
Try delete, when trying to delete not work with me. @SEAbdulbasit